src/components/BudgetOverviewCard/BudgetItem.graphql

```diff
@@ -0,0 +1,14 @@
+fragment BudgetItem on Budget {
```

Comment:
Could you also rename this? This isn't a list item so I didn't want to get that confused. It really is just an overview of the budget.
```suggestion
fragment BudgetOverview on Budget {
```
And it makes sense for the fragment to be similar to the component name.